form/unit_testing.rst

```diff
@@ -185,10 +185,10 @@ allows you to return a list of extensions to register::
 
         protected function getExtensions()
         {
-            $this->validator = $this->createMock(ValidatorInterface::class);
+            $validator = $this->createMock(ValidatorInterface::class);
             // use getMock() on PHPUnit 5.3 or below
-            // $this->validator = $this->getMock(ValidatorInterface::class);
-            $this->validator
+            // $validator = $this->getMock(ValidatorInterface::class);
+            $validator
                 ->method('validate')
                 ->will($this->returnValue(new ConstraintViolationList()));
             $validator
```

Comment:
Indeed, luckily we already have a pull request suggesting exactly this change (see #8013). So this will be fixed soon. That's why I'm gonna close here. But thank you for making us aware of this issue @hthtung90. Looking forward to see you soon again with some other changes. :)